sktime/forecasting/compose/_pipeline.py

```diff
@@ -1199,6 +1199,11 @@ class ForecastX(BaseForecaster):
     columns : None, or pandas compatible index iterator (e.g., list of str), optional
         default = None = all columns in X are used for forecast
         columns to which `forecaster_X` is applied
+    fit_behaviour : str, one of "ues_gt", "ues_forecast", optional,
+        default = "ues_gt"
```

Comment:
I'm not sure if `use_gt` is immediately obvious. I'll suggest `use_actual` or something similar.

Reply:
there are typos too. Agree with improving naming convention

Reply:
use_actual is a good idea.